### DISCUSSION TITLE
Integration scATAC with scRNAseq not multiomic

### DISCUSSION THREAD

**Discussion by yamihn** [stuart-lab/signac#1723] [Q&A]
Hi @timoast , I have 4 scATAC fragments (2ko and 2wt) and I would to process and integrate with 4 scRNAseq (2ko and 2wt) from **the same experiment but NOT MULTIOMIC** 
Conditions correspond but the cells seems to be differents. (I don't find scRNA Cell names in the respective scATAC fragment).
Which kind how approach do you suggest to me?
I am not sure if I can integrate not multiomic sample.
Best
Olga

**Comment by timoast** [comment]
Hi @yamihn 

Yes, it's expected that the cells names will be different for separate scATAC and scRNA experiments, as the actual cells profiled are different cells (in multiome the two measurements come from the same individual cell).

We have some examples of how to integrate scRNA-seq and scATAC-seq data for the purpose of cell type annotation on the Signac website: https://stuartlab.org/signac/articles/pbmc_vignette#integrating-with-scrna-seq-data
